lib/OpenQA/WebAPI/Auth/OpenID.pm

```diff
@@ -90,8 +90,8 @@ sub auth_response {
         return (error => 'Got response on http but https is forced. MOJO_REVERSE_PROXY not set?');
     }
 
-    while (my ($k, $v) = each %params) {
-        $params{$k} = URI::Escape::uri_unescape($v);
+    foreach my $key (sort keys %params) {
```

Comment:
I'd definitely remove the sort here. It will only slow things down with no additional benefit.